js/index.js

```diff
@@ -83,20 +122,29 @@ function tryAgain(e) {
     const retryButton = document.querySelector('.retry-button')
     const targetKeyboard = document.querySelector('.keyboard')
     const allTiles = document.querySelectorAll('.game > * > span')
+    const allKeys = document.querySelectorAll('.key')
 
     // Retry process
     gameState.attemptCounter = 0
     gameState.tryCounter++
     gameState.expectedWord = e[gameState.tryCounter].toUpperCase().split("")
+
     allTiles.forEach(function (tiles) {
         tiles.textContent = ''
+        tiles.classList.remove('correct-letter')
+        tiles.classList.remove('incorrect-letter')
+        tiles.classList.remove('correct-position')
```

Comment:
You can pass multiple classes into remove to make this more DRY

Reply:
yep